diff --git a/node_modules/@opentelemetry/context-async-hooks/build/src/AsyncHooksContextManager.js b/node_modules/@opentelemetry/context-async-hooks/build/src/AsyncHooksContextManager.js
index 28b248a..0cbe43b 100644
--- a/node_modules/@opentelemetry/context-async-hooks/build/src/AsyncHooksContextManager.js
+++ b/node_modules/@opentelemetry/context-async-hooks/build/src/AsyncHooksContextManager.js
@@ -16,7 +16,12 @@
  */
 Object.defineProperty(exports, "__esModule", { value: true });
 exports.AsyncHooksContextManager = void 0;
-const asyncHooks = require("async_hooks");
+let asyncHooks;
+try {
+    asyncHooks = require("async_hooks");
+} catch (e) {
+    // ignore, not in node
+}
 const AbstractAsyncHooksContextManager_1 = require("./AbstractAsyncHooksContextManager");
 class AsyncHooksContextManager extends AbstractAsyncHooksContextManager_1.AbstractAsyncHooksContextManager {
     constructor() {
diff --git a/node_modules/@opentelemetry/context-async-hooks/build/src/AsyncLocalStorageContextManager.js b/node_modules/@opentelemetry/context-async-hooks/build/src/AsyncLocalStorageContextManager.js
index e83273e..d266a2b 100644
--- a/node_modules/@opentelemetry/context-async-hooks/build/src/AsyncLocalStorageContextManager.js
+++ b/node_modules/@opentelemetry/context-async-hooks/build/src/AsyncLocalStorageContextManager.js
@@ -16,7 +16,12 @@
  */
 Object.defineProperty(exports, "__esModule", { value: true });
 exports.AsyncLocalStorageContextManager = void 0;
-const async_hooks_1 = require("async_hooks");
+let async_hooks_1;
+try {
+    async_hooks_1 = require("async_hooks");
+} catch (e) {
+    // ignore, not in node
+}
 const AbstractAsyncHooksContextManager_1 = require("./AbstractAsyncHooksContextManager");
 /**
  * This class provides a context manager for Node.js environments that support

